src/pages/home/profile/Profile.tsx

```diff
@@ -161,127 +185,204 @@ const Profile: FunctionComponent = () => {
+                        justifyContent: "space-between",
+                        rowGap: 2,
+                      } }
+                    >
+                      <TextInputField
+                        label="FIRST NAME"
+                        name="firstName"
+                        placeholder="First name"
+                        type="text"
+                      />
+                      <TextInputField
+                        label="LAST NAME"
+                        name="lastName"
+                        placeholder="Last name"
+                        type="text"
+                      />
+                    </Stack>
+                    <TextInputField
+                      disabled={ true }
+                      label="PRIMARY EMAIL"
+                      name="email"
+                      placeholder="john@mail.com"
+                      type="email"
+                    />
+                  </Stack>
+                  <Stack rowGap={ 2 }>
+                    <Typography variant="h4" fontWeight="700">
+                      OTHER SETTINGS
+                    </Typography>
+                    <SwitchInputField
+                      name="companyIpRights"
+                      title="DO YOU HAVE A COMPANY?"
+                      description={
+                        "If your IP Rights are held under your Company, please select this option."
+                      }
+                    >
+                      <Stack
+                        alignItems="center"
+                        columnGap={ 1.5 }
+                        flexDirection="row"
+                        mt={ 2 }
+                      >
+                        <img
+                          alt="TODO: Replace this with reusable img component"
+                          src={ pictureUrl }
+                          style={ { height: "60px", width: "60px" } }
+                        />
+                        <TextInputField
+                          aria-label="Your company name"
+                          name="companyName"
+                          placeholder="Your company name"
+                          type="text"
+                        />
+                      </Stack>
+                    </SwitchInputField>
+                  </Stack>
+                </Stack>
+
+                <HorizontalLine
+                  sx={ {
+                    maxWidth: ["340px", "340px", "700px"],
+                    mx: ["auto", "auto", "unset"],
+                    my: 5,
+                  } }
                 />
-              </Stack>
+                <Stack
+                  sx={ {
+                    columnGap: 2,
+                    flexDirection: [null, null, "row"],
+                    mt: 5,
+                    rowGap: 2,
+                  } }
+                >
+                  <Button
+                    disabled={ !dirty }
+                    onClick={ handleReset }
```

Comment:
I made this Button `type="reset"` on the Settings page, does that get handled differently than a  `handleReset` passed through Formik?

I also have this button `color="music"` per the Figma design.